src/main/java/frc/robot/commands/ledcommands/ChangeLEDToRed.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.ledcommands;
+
+import frc.robot.subsystems.LEDSubsystem;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** An example command that uses an example subsystem. */
+public class ChangeLEDToRed extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+
+  private final LEDSubsystem ledSubsystem;
+
+  public ChangeLEDToRed(LEDSubsystem ledSubsystem) {
+    this.ledSubsystem = ledSubsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(ledSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    System.out.println("red leds have succesfully executed");
```

Comment:
TODO: Remove print statements that say the led has succesfully executed for all change color commands.

Reply:
Done